src/HelpText/widget/HelpText.js

```diff
@@ -89,18 +99,18 @@ mendix.widget.declare('HelpText.widget.HelpText', {
 	
 	showHelp : function(show, clicked) {
 		if (show || this.helpvisible) {
-			var coords = dojo.coords(this.imgNode, true);
+			var coords = domGeom.position(this.imgNode, true);

```

Comment:
Please double check `domGeom.position` is equal to  `dojo.coords`

Especially when your are  scrolled